lib/travis/config/defaults.rb

```diff
@@ -43,8 +43,8 @@ def fallback_logs_api_auth_token
             amqp:                 { username: 'guest', password: 'guest', host: 'localhost', prefetch: 1 },
             closeio:              { key: 'key' },
             gdpr:                 {},
-            database:             { adapter: 'postgresql', database: "travis_#{Travis.env}", encoding: 'unicode', min_messages: 'warning', variables: { statement_timeout: 10_000 } },
-            db:                   { max_statement_timeout_in_seconds: 15, slow_host_max_statement_timeout_in_seconds: 60},
+            database:             { adapter: 'postgresql', database: "travis_#{Travis.env}", encoding: 'unicode', min_messages: 'warning', variables: { statement_timeout: 45_000 } },
```

Comment:
keep master + the timeouts need to be from env

Reply:
Done